superchain/extra/addresses/addresses.json

```diff
@@ -146,6 +146,9 @@
     "SystemConfigProxy": "0x034edD2A225f7f429A63E0f1D2084B9E0A93b538",
     "ProxyAdminOwner": "0xDEe57160aAfCF04c34C887B5962D0a69676d3C8B",
     "SystemConfigOwner": "0xfd1D2e729aE8eEe2E146c033bf4400fE75284301",
+    "DisputeGameFactoryProxy": "0x05F9613aDB30026FFd634f38e5C4dFd30a197Fa1",
+    "AnchorStateRegistryProxy": "0x1874A5CCA34eBCe29aDf1ebdd7fD2725fe184838",
+    "DelayedWETHProxy": "0x9bD56B189234FE25CF3547c4343e50C3259e383a",
```

Comment:
We also need to include the implementation addresses for these, and the updated portal implementation, in https://github.com/ethereum-optimism/superchain-registry/blob/main/superchain/implementations/networks/sepolia.yaml. Then we can close #135 and replace it with this one